test/spec/helpers/assertUrlStorage.js

```diff
@@ -8,9 +8,9 @@ module.exports = async function assertUrlStorage(urlStorage) {
   const key2 = await urlStorage.addUpload("fingerprintA", { id: 2 });
   const key3 = await urlStorage.addUpload("fingerprintB", { id: 3 });
 
-  expect(key1.startsWith("tus::fingerprintA::")).toBe(true);
-  expect(key2.startsWith("tus::fingerprintA::")).toBe(true);
-  expect(key3.startsWith("tus::fingerprintB::")).toBe(true);
+  expect(/^tus::fingerprintA::/.test(key1)).toBe(true);
+  expect(/^tus::fingerprintA::/.test(key2)).toBe(true);
+  expect(/^tus::fingerprintB::/.test(key3)).toBe(true);
```

Comment:
Instead of adding a polyfill for `startsWith`, I just used an older way of doing the same thing. This should prob continue to be done for test code, else we might have more polyfills in the tests than we expect users to have, and we could accidentally depend on them in the library code.

Reply:
Absolutely agree, you are right! :+1: